app/routes/bdb/BdbRoute.js

```diff
@@ -24,11 +25,16 @@ const mapStateToProps = (state, props) => ({
   fetching: state.companies.fetching
 });
 
-const mapDispatchToProps = {
-  editSemesterStatus,
-  addSemesterStatus,
-  addSemester
-};
+function mapDispatchToProps(dispatch) {
+  return {
```

Comment:
Why are you doing this?

Reply:
To get `push`, which needs to be dispatched, right?

Reply:
Maybe I could have simply added `push` to the simpler version? Didn't even strike me as an opportunity